### ISSUE TITLE
No sound after getting back Steam Deck from sleep

### ISSUE THREAD

**Issue by Pruebaa84** [ValveSoftware/SteamOS#1376]
#### Your system information

* Steam client version: 1705108172
* SteamOS version: 3.5.7
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
Everytime that I put the Steam Deck to sleep, it comes back with no sound after waking it. It makes no difference if during a game or not, there is no sound even when I out it to sleep in the Steam menu. Only thing I did back then was to change the Proton Version to load Baldurs Gate 3, I never even accessed Desktop mode till yesterday. Also a factory reset didn't help on this issue. I presented this issue in the SteamCommunity forums and they redirected me here after seeinf my journalct.

#### Steps for reproducing this issue:

1. Just put the Steam Decdk to sleep.
2. Wake it up and no sound.

And this is the journalct from this last day to ilustrate further the issue:

-- Boot c76475a797ab414584ff5c8cb481824c --
Feb 04 00:33:50 steamdeck systemd[864]: Started PipeWire Multimedia Service.
Feb 04 00:33:50 steamdeck pipewire[1009]: mod.rt: Can't find xdg-portal: (null)
Feb 04 00:33:50 steamdeck pipewire[1009]: mod.rt: found session bus but no portal
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_add_plugin(): warning: Duplicate plugin [<https://github.com/werman/noise-suppression-for-voice>](http://&lt;https//github.com/werman/noise-suppression-for-voice&gt;)
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_add_plugin(): warning: ... found in file:///usr/lib/lv2/rnnoise_mono.lv2/
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_add_plugin(): warning: ... and file:///usr/lib/lv2/rnnoise_stereo.lv2/ (ignored)
Feb 04 00:33:50 steamdeck pipewire[1009]: error: failed to open file [/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl](http://usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl) (Not a directory)
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_load_file(): error: Error loading file [file:/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl'](http://file/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl')
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_load_bundle(): error: Error reading [file:/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl](http://file/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl)
Feb 04 00:33:50 steamdeck pipewire[1009]: error: failed to open file /usr/lib/lv2/svg/manifest.ttl (No such file or directory)
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_load_file(): error: Error loading file file:///usr/lib/lv2/svg/manifest.ttl'
Feb 04 00:33:50 steamdeck pipewire[1009]: lilv_world_load_bundle(): error: Error reading file:///usr/lib/lv2/svg/manifest.ttl
Feb 04 00:34:18 steamdeck pipewire[1009]: spa.alsa: hw:acp5x,1: (0 missed) snd_pcm_avail after recover: Broken pipe
Feb 04 00:39:51 steamdeck systemd[864]: Stopping PipeWire Multimedia Service...
Feb 04 00:39:51 steamdeck systemd[864]: Stopped PipeWire Multimedia Service.
Feb 04 00:39:51 steamdeck systemd[864]: pipewire.service: Consumed 1.353s CPU time.
-- Boot 154e428e4ef84e9b882833e901163dfa --
Feb 04 00:40:48 steamdeck systemd[861]: Started PipeWire Multimedia Service.
Feb 04 00:40:48 steamdeck pipewire[1056]: mod.rt: Can't find xdg-portal: (null)
Feb 04 00:40:48 steamdeck pipewire[1056]: mod.rt: found session bus but no portal
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_add_plugin(): warning: Duplicate plugin [<https://github.com/werman/noise-suppression-for-voice>](http://&lt;https//github.com/werman/noise-suppression-for-voice&gt;)
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_add_plugin(): warning: ... found in file:///usr/lib/lv2/rnnoise_mono.lv2/
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_add_plugin(): warning: ... and file:///usr/lib/lv2/rnnoise_stereo.lv2/ (ignored)
Feb 04 00:40:48 steamdeck pipewire[1056]: error: failed to open file [/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl](http://usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl) (Not a directory)
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_load_file(): error: Error loading file [file:/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl'](http://file/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl')
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_load_bundle(): error: Error reading [file:/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl](http://file/usr/lib/lv2/valve_audio_dsp_version.txt/manifest.ttl)
Feb 04 00:40:48 steamdeck pipewire[1056]: error: failed to open file /usr/lib/lv2/svg/manifest.ttl (No such file or directory)
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_load_file(): error: Error loading file file:///usr/lib/lv2/svg/manifest.ttl'
Feb 04 00:40:48 steamdeck pipewire[1056]: lilv_world_load_bundle(): error: Error reading file:///usr/lib/lv2/svg/manifest.ttl
Feb 04 00:42:45 steamdeck pipewire[1056]: spa.alsa: hw:acp5x,1: (0 missed) snd_pcm_avail after recover: Broken pipe

**Comment by dangerousdonut** [comment]
I'm seeing this a lot in 3.6.3. Restarting fixes it.

**Comment by gebba** [comment]
Just updating to say I get this 100% of the time when I put the deck to sleep. No bets, with beta, done various factory resets and bios resets, no change. Been this way for months now... Full power down and boot up always solves it, but a restart does not.

**Comment by KonstantinAlexeevich** [comment]
I have same problem.
I tried:

- Toggle "Use as Wake Move" in customization tab
- Updates from beta and preview branches
- Factory Reset
- "Re-image Steam Deck" from usb [here](https://help.steampowered.com/en/faqs/view/1B71-EDF2-EB6D-2BB3)

All of this points not working for me. It's so sad.

**Comment by Pruebaa84** [comment]
Yeah, in the end I sent it to repair back then. After their technical helpdesk were keeping me in loops trying one thing and another. I will close this topic.

**Comment by gebba** [comment]
Is there going to be any updates on this? As this started happening I've really used my Steam Deck a lot less, sleep mode instant pickup was a big use case for me. And I've seen people have this issue on OLEDs too, so getting another one isn't really likely to solve anything.

**Comment by xiaolin0520** [comment]
> Is there going to be any updates on this? As this started happening I've really used my Steam Deck a lot less, sleep mode instant pickup was a big use case for me. It happens for all games. And I've seen people have this issue on OLEDs too, so getting another one isn't really likely to solve anything.

I have a dual system installed and the sound works fine under win11, but no sound under steamOS. So it should be an Arch Linux sound driver issue. I fixed it by constantly restarting the Linux sound system. I don't remember the exact code, I fixed it by asking chatgpt for the code.

**Comment by gebba** [comment]
> > Is there going to be any updates on this? As this started happening I've really used my Steam Deck a lot less, sleep mode instant pickup was a big use case for me. It happens for all games. And I've seen people have this issue on OLEDs too, so getting another one isn't really likely to solve anything.
> 
> I have a dual system installed and the sound works fine under win11, but no sound under steamOS. So it should be an Arch Linux sound driver issue. I fixed it by constantly restarting the Linux sound system. I don't remember the exact code, I fixed it by asking chatgpt for the code.

Seems weird for it to be a driver issue though? Then it should affect a larger set of people? Feels like it has to be related to something we did? I kind of feel like it started happening to me when I opted into the very hyped Steam OS 3.5 preview... but it's been so long I cant be sure. I finally bothered to send a support ticket again today... hoping to hear something from them.

**Comment by gebba** [comment]
I still have this issue 10 months later. But recently found a reddit user saying that if you hold the volume up or down button when waking it from sleep it comes back out of sleep with audio functioning.

I just tried, and it works. Wth? It even works when i have awakened the device without holding a volume button to sleep it again, and re-awaken it with a button held to get audio backif it was lost in a previous wake. Thats bonkers.

This if anything should confirm its a software issue somehow? Or firmware for the audio board somehow? Can we get more eyes on this?

And this is with all the latest stable steamOS updates still.

**Comment by dannycollins84-code** [comment]
I have had this issue latest build all upto date, Ive noticed that its not all the time last games I played and this happened was titanfall 2 to get sound back you have to restart system

**Comment by gebba** [comment]
> I have had this issue latest build all upto date, Ive noticed that its not all the time last games I played and this happened was titanfall 2 to get sound back you have to restart system 

I dont think this is necessarily the same issue entirely. This issue affects every game, and even Steam UI game mode sounds itself for me. Even a restart does not help, I have to do a full shutdown, and then start again to get audio. Or hold the volume button as i wake the device apparently.

**Comment by AlexSnowLeo** [comment]
I have a similar problem, but it only manifests itself when the Deck goes into sleep mode on its own. And it seems more often when the game was being downloaded. 
SteamOS Version: 3.7.17

**Comment by st235** [comment]
> I have had this issue latest build all upto date, Ive noticed that its not all the time last games I played and this happened was titanfall 2 to get sound back you have to restart system

Happens to me with `3.7.17` update as well

**Comment by tvjcbhghgb-ship-it** [comment]
Same here, after come back from sleep my deck lost not only the sound ,but also the WIFI connection, a reboot always solve the problem,but you can't always expect to reboot your device everytime.

os 3.7.17

**Comment by Haagenati** [comment]
I am seeing the same issue on 3.7.17

No sound after waking from sleep and also no wifi, while troubleshooting I tried switching into desktop mode and no audio outputs were detected and no wifi available at all. Restarting the devices is the only solution I've found.

**Comment by pr0tonion** [comment]
> I am seeing the same issue on 3.7.17
> 
> No sound after waking from sleep and also no wifi, while troubleshooting I tried switching into desktop mode and no audio outputs were detected and no wifi available at all. Restarting the devices is the only solution I've found. 

Same issue here. Restart fixed it, but this Should be looked at

**Comment by jwavoet** [comment]
This issue started a couple of days ago on my OLED running 3.7.19. It is resolved after a reboot. It does seem to not occur immediately after reboot going into sleep and waking up again.

**Comment by bastiHST90** [comment]
I had that problem on my steam deck OLED (always stable steamos) like 2 times in 2 1/2 years. But 2 times today.

**Comment by mefistos** [comment]
Same issue happening on my Steamdeck running 3.7.20 OS version, wish this was liiked into since it has been happening for years now. Whenevr I lose audio the Wifi also goes out.

**Comment by bendover28** [comment]
I am also having this issue. All sound and occasionally wifi cut out after waking from sleep. Only restarting fixes it. Updating to latest stable version of SteamOS didn't fix it.

**Comment by rovermicrover** [comment]
>  [I]f you hold the volume up or down button when waking it from sleep it comes back out of sleep with audio functioning.
> 
> I just tried, and it works. Wth? It even works when i have awakened the device without holding a volume button to sleep it again, and re-awaken it with a button held to get audio backif it was lost in a previous wake. Thats bonkers.

An update in May 2026 holding the volume buttons still work! Annoying but much quicker than restart!
